apps/central-scan/frontend/src/screens/settings_screen.tsx

```diff
@@ -98,10 +99,10 @@ export function SettingsScreen({
         />
       </ButtonRow>
       {!canUnconfigure && (
-        <P>
-          <Icons.Warning color="warning" /> You must &quot;Save Backup&quot;
-          before you may unconfigure this machine.
-        </P>
```

Comment:
<img width="436" alt="Screen Shot 2024-10-13 at 11 45 24 PM" src="https://github.com/user-attachments/assets/b988a01a-59af-49e3-994e-7a51c1d9b36b">
<img width="619" alt="Screen Shot 2024-10-13 at 11 40 15 PM" src="https://github.com/user-attachments/assets/b516754f-0f7f-415c-9a2d-9394cec335d7">

Reply:
Since I am using the caption for the text under the the toggle, it made sense to make them the same size.